runtime/compiler/optimizer/IdiomTransformations.cpp

```diff
@@ -6474,9 +6472,9 @@ makeMemCpyCharToByteGraph(TR::Compilation *c, int32_t ctrl)
    }
 
 
-//////////////////////////////////////////////////////////////////////////
-//////////////////////////////////////////////////////////////////////////
-//////////////////////////////////////////////////////////////////////////
+
+
+
```

Comment:
Lines that contain only slashes will be ignored by Doxygen, so there's no need to change these lines. May I ask you to change them back to the following?

```
//////////////////////////////////////////////////////////////////////////
//////////////////////////////////////////////////////////////////////////
//////////////////////////////////////////////////////////////////////////
```